ERCS/erc-draft-wallet_transmitCrossChainIntents.md

```diff
@@ -0,0 +1,192 @@
+---
+title: wallet_transmitCrossChainIntents Method
+description: Standard Cross-Chain Intent Parsing Method For EVM Wallet
+author: ZeroKPunk (@ZeroKPunk), 0xbbPizza (0xbbPizza)
+discussions-to: <URL>
```

Comment:
Please create a discussions topic in Eth Magicians with a link to this PR: https://ethereum-magicians.org/c/ercs/57